src/models/BonusNumber.js

```diff
@@ -0,0 +1,58 @@
+import LOTTO_CONSTANTS from '../constants/lottoConstatns.js';
+import ERROR_MESSAGES from '../constants/messages/errorMessages.js';
+
+class BounsNumber {
+  #winningNumbers;
+  #bonusNumber;
+
+  constructor(winningNumbers, bonusNumber) {
+    this.#winningNumbers = winningNumbers;
+    this.#validateBonusNumber(bonusNumber);
+    this.#bonusNumber = Number(bonusNumber);
+  }
```

Comment:
밑에 코드에서 `Number(bonusNumber)`가 반복되어 나타나고 있어요! 그러면 애초에 `Number`로 타입을 변환 후, `validateBonusNumber`에 넘겨주면 `Number()`를 반복적으로 사용하지 않아도 된다고 생각합니다!!

```suggestion
  constructor(winningNumbers, bonusNumber) {
    bonusNumber = Number(bonusNumber)
    this.#winningNumbers = winningNumbers;
    this.#validateBonusNumber(bonusNumber);
    this.#bonusNumber = bonusNumber;
  }
```

Reply:
좋은 제안 감사합니다!